includes/gateway/class-omise-payment-fpx.php

```diff
@@ -104,4 +104,15 @@ public function charge( $order_id, $order ) {
 			'metadata'    => $metadata
 		) );
 	}
+
+	/**
+	 * Get icons
+	 *
+	 * @see WC_Payment_Gateway::get_icon()
+	 */
+	public function get_icon() {
+		$icon =  Omise_Image::get_image( 'fpx.svg', 'FPX', 60, 60 );
```

Comment:
Instead of passing 60 directly, it would be better if we assign it to a variables and use those instead. It would be easy to understand what the value represent.